awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
```

Comment:
JDetails for `ui_next` looks fine:
![cred_jdetails](https://user-images.githubusercontent.com/9753817/87680656-b1578900-c74b-11ea-8c50-144a781f251c.gif)